lib-cloudsdk/src/main/java/com/facebook/openwifi/cloudsdk/UCentralClient.java

```diff
@@ -690,6 +690,19 @@ public RRMDetails getProvInventoryRRMDetails(String serialNumber) {
 		try {
 			return gson.fromJson(response.getBody(), RRMDetails.class);
 		} catch (JsonSyntaxException e) {
+			// catch strings like "no", "inherit", "invalid" (???)
+			JSONObject respBody;
+			try {
+				respBody = new JSONObject(response.getBody());
+				respBody.getString("rrm");
+				logger.error(
+					"RRMDetails returned unexpected string body: {}",
+					respBody
+				);
+				return null;
+			} catch (JSONException e2) { /* ignore and fall through */}
```

Comment:
Add some error logs in this catch block?

Reply:
This block was only checking if the body was a different type which isn't expected anyway, which is why we ignore any error here.